### PR TITLE
Refine helm charts of graphscope store

### PR DESCRIPTION
**Pull request by siyuan0322**

### CODE REVIEW

File: charts/graphscope-store/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+annotations:
+  category: Database
+apiVersion: v2
+appVersion: 0.4.1
+description: Chart to create a Highly available MySQL cluster
```
Comment (yecol) [inline charts/graphscope-store/Chart.yaml:5]: revise the desc.

File: charts/graphscope-store/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# GraphScope Store
+
+[GraphScope Store](https://github.com/alibaba/GraphScope/tree/main/interactive_engine/src/v2/src/main) is a new disk-based row-oriented multi-versioned persistent graph store.
+
+
+## TL;DR
+
+```bash
+$ helm repo add graphscope https://graphscope.oss-cn-beijing.aliyuncs.com/store/charts/
+$ helm install my-release graphscope/graphstore
```
Comment (yecol) [inline charts/graphscope-store/README.md:10]: could you make the name consistent? maybe graphscope-store? as you describe in the Chart.yml

File: charts/graphscope-store/README.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+## Introduction
+
+This chart bootstraps a [GraphScope Store](https://github.com/alibaba/GraphScope/tree/main/interactive_engine/src/v2/src/main) cluster deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 3.1.0
+- PV provisioner support in the underlying infrastructure
```
Comment (yecol) [inline charts/graphscope-store/README.md:21]: PersistentVolume(PV) provisioner ....

File: charts/graphscope-store/README.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+```bash
+$ helm install my-release -f values.yaml graphscope/graphstore
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+
+## Persistence
+
+The [GraphScope Store](https://github.com/alibaba/GraphScope/tree/main/interactive_engine/src/v2/src/main) image stores the GraphScope Store data at the `/opt/graphscope-data` and configurations at the `/opt/graphscope-meta` path of the container.
```
Comment (yecol) [inline charts/graphscope-store/README.md:96]: maybe use a reasonable path?

**Review by yecol — CHANGES_REQUESTED** [review]

**Review by sighingnow — APPROVED** [review]

**Review by sighingnow — APPROVED** [review]

File: charts/graphscope-store/README.md (hunk 4 above)
Comment (siyuan0322) [reply charts/graphscope-store/README.md]: Any suggestions?

Comment (yecol) [reply charts/graphscope-store/README.md:99]: /mnt

Comment (yecol) [reply charts/graphscope-store/README.md:99]: /var

**Review by lidongze0629 — APPROVED** [review]

**Pull request merged**